package.json

```diff
@@ -80,7 +80,7 @@
     "sinon": "^8.0.1"
   },
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=10.18.1"
```

Comment:
I am wondering why `10.18.1` is chosen as the minimum support version? Are we depending on the API provided since `10.18.1`?

AFAIK all `node >= 10.9 < 11` shares the same V8 major and `node >= 10.12` has recursive `fs.mkDir` support which we may benefit from.

Reply:
[`10.18.0`](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2019-12-17-version-10180-dubnium-lts-mylesborins) includes a critical security fix.

[`10.18.1`](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2020-01-09-version-10181-dubnium-lts-bethgriggs) has timezone fix, maybe related to https://github.com/hexojs/hexo/issues/4239?

Reply:
I don't see a strong reason limit to `>=10.18.1` because

1) Security fix will always be patched as long as node.js 10 is in active LTS but apparently we will not bump `engines` to latest patches

2) On the timezone fix, did you refer to the `tzdata` updates? It seems to me that this is a routine thing but I may be wrong.

The `engines` requirements should be specified as a minimal versions to run `hexo` successfully. So typically it is bound to specific V8 versions / Node.js API, which means `hexo` can not even run on Node.js versions lower than `10.x.y`.

That said, it does not mean that we recommend users to _stay_ at the lowest version. The best practice is to use LTS and to update to the latest minors.

Reply:
@JLHwung @curbengh 

Well, what about Node.js 10.13.0 then? It is the first LTS version of Node.js 10 and could be used as the minimum required version then.

Reply:
FYI, https://github.com/hexojs/hexo-fs/pull/55 required Node.js version to be higher than 10.12.0.